src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -68,6 +68,7 @@ public static void main(String[] args) throws Exception {
                 :Triplet.fromCurrentOS();
 
         String expected  = System.getProperty("expected");
+        boolean verbose = System.getProperty("verbose") != null;
```

Comment:
Should we require `-Dverbose=true`? For other booleans we do that

Reply:
I wouldn't do that yet, as we might go for different levels instead (-Dverbose=INFO) .
-Dverbose keeps the options open.

Reply:
Makes sense